composer.json

```diff
@@ -36,6 +36,7 @@
         "zendframework/zend-httphandlerrunner": "^1.0.1"
     },
     "require-dev": {
+        "filp/whoops": "^2.3",
```

Comment:
Looks like _what_ is included in 2.3.1?

Reply:
`handleUnconditionally()` has been present in Whoops since 1.1.0 _(I think)_ [here's the relevant commit](https://github.com/filp/whoops/commit/98140ba126db66805717b941a95b49cd5cba4d21) - As a dev constraint, would it be better to use something like `^2.1` as that's the minimum to get stuff happening in Expressive??